src/recipe.js

```diff
@@ -19,7 +19,8 @@ class Recipe {
         }
       })
     });
-    return costCounter;
+    let costInDollars = (costCounter/100).toFixed(2);
+    return Number(costInDollars);
```

Comment:
I think we can use a map instead of a forEach when searching through an ingredient.  Since it is the same length output of the ingredient, then we can use Map and set it to a variable and then we can use Find on the new array.

Reply:
Or we could use a reduce combined with find to get the specific ingredients and then reduce the total cost in cents once we have all of those items found.

Reply:
Great ideas!